pre-check/v1.x/check.sh

```diff
@@ -587,6 +587,51 @@ check_backup_target()
     echo -e "\n==============================\n"
 }
 
+# This throws a warning if the minimum number of copies for a backing image is less than the default (3) 
+# Or if the minimum number of copies is set to '0' - it failes the test. VMs won't start if the backing image isn't available after upgrade. 
+check_images()
+{
+    log_info "Starting Longhorn Backing Images check..."
+    log_verbose "NOTE: This test will throw a warning if less than the default value and fail if minimum number of copies is set to 0."
+    # Use a file to store the clean state becuase we can't set the global variable inside the piped scope
+    # The file is removed in the event of a failure
+    clean_state=$(mktemp)
+
+    backingImageList=$(kubectl get backingImage -A  -o yaml | yq -r '.items[] | .metadata.name ')
+    for backingImage in $backingImageList ; do
+        minCopies=$(kubectl -n longhorn-system get backingImage $backingImage -o yaml | yq -r '.spec.minNumberOfCopies ')
+        if [[ $minCopies -eq 0 ]]; then
+            log_info "WARN: $backingImage has no minimum number of copies set!"
+            rm -f $clean_state
+            sleep 0.5
+            continue
+        elif [[ $minCopies -lt 3 ]]; then
+            log_info "Warning: Backing image $backingImage has a minimum number of copies set to $minCopies. It is strongly encouraged to set this number to be at least 3 or higher. "
+            if [ ! -e "$warn_state" ]; then
+                warn_state=$(mktemp)
+            fi 
+        else
+            log_verbose "$backingImage has $minCopies minimum copies set."
+        fi
+    done
```

Comment:
I think with a bit of `kubectl` magic, we should be able to retrieve these two lists with:

```sh
➜ kubectl get backingImage -A -ojsonpath='{.items[?(@.spec.minNumberOfCopies==0)].metadata.name}'

➜ kubectl get backingImage -A -ojsonpath='{.items[(@.spec.minNumberOfCopies<3)].metadata.name}'
```

And likely, we won't even need the temp files.